nmdc_runtime/test.Dockerfile

```diff
@@ -34,4 +34,6 @@ RUN chmod +x wait-for-it.sh
 # Best practices: Prepare for C crashes.
 ENV PYTHONFAULTHANDLER=1
 
-ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest", "-x"]
\ No newline at end of file
+# disable stop after first failure to run all tests
+ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest"]
+#ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest", "-x"]
```

Comment:
FYI to other reviewers:
- The `-x` option told pytest to stop after the first error

Reference: https://docs.pytest.org/en/6.2.x/usage.html#stopping-after-the-first-or-n-failures